src/main/java/core/basesyntax/fruitshop/ValidatorImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.fruitshop;
+
+public class ValidatorImpl implements Validator {
+    private static final String IS_NUMERIC = "[0-9]+";
+    private static final String VALID_ACTION = "[bspr]";
+    private final FruitChecker fruitChecker = new FruitCheckerImpl();
```

Comment:
it's better to receive fuitChecker object from constructor, so you will be able to switch impls easily